lib/kernel/src/code_server.erl

```diff
@@ -510,10 +523,17 @@ add_loader_path(IPath0,Mode) ->
 	    PrimP = patch_path(PrimP0),
 	    IPath = patch_path(IPath0),
 
-            P = exclude_pa_pz(PrimP,Pa,Pz),
-            Path0 = strip_path(P, Mode),
-            Path = add(Path0, IPath, []),
-            add_pa_pz(Path,Pa,Pz)
+            Path0 = (PrimP -- Pa) -- Pz,
```

Comment:
I can send a separate pull request for this change, if desired, but I realized that `exclude_pa_pz/3` is pretty much a more expensive and verbose way of doing exactly this.